src/data/markdown/docs/01 guides/05 Testing Guides/02 Load testing websites.md

```diff
@@ -3,115 +3,99 @@ title: "Load testing websites"
 
-On the other side, you have to identify what your users will do regularly. You could look at your analytics tools to find the most frequent user journeys.
+![](./Frontend-Backend-LoadTesting.png)
 
-With this information, it’s time to analyze the frequency of usage, business value, performance risks, and any other critical performance aspect of your organization to decide what to load test first.
 
-As a rule of thumb, <a href="https://k6.io/our-beliefs#simple-testing-is-better-than-no-testing">Simple testing is better than no testing</a>:
+## When should I load test a website?
 
-> Start small and simple, make sure you get something out of the testing first, then expand the test suite and add more complexity until you feel that you’ve reached the point where more effort spent on realism will not give enough return on your invested time.
+In short, when you are concerned about the **availability and scalability of your website**. 
 
+If you have very low number of users, backend performance won't likely be an issue. Spend your time optimizing the frontend. 
 
-Once you adopt performance testing in your development lifecycle, your performance test suite will grow over time. You will add and update performance tests based on the evolution of your application, new performance areas, bugs, and the maturity of your team and processes, amongst other factors.
+But as the number of user grows, you should start spending more time improving and testing the performance of your backend. In the [performance golden rule](https://www.stevesouders.com/blog/2012/02/10/the-performance-golden-rule/), Steve Souders pointed out that:
 
-To sum up, start small, test frequently and iterate, iterate, iterate.
+> If you’re worried about availability and scalability, focus on the backend.
 
-## Should I load test my website in production or pre-production?
+### Browser metrics and load testing
 
-We recommend to <a href="https://k6.io/our-beliefs#load-test-in-a-pre-production-environment">Load test in a pre-production environment</a> that mimics the production environment as closely as possible.
+When testing how your system behaves with some concurrent users, browser metrics are, in most cases, less useful. The reason is that each browser runs independently of the number of users visiting your website.  Hence, a load test will not impact browser metrics like rendering time.
 
-Testing regularly in a pre-production environment allows assessing the performance of your website over time and catch performance regressions before they reach production.
 
-Load testing in pre-production allows breaking your website without worrying about the interruption of your service.  Load testing in production is risky, but if your processes and team are mature, you could run your load tests in production as part of your <a href="https://principlesofchaos.org/">chaos experiments</a>. Sometimes it is the best way to get a picture of the real world.
+## How to load test a website?
```

Comment:
`How to load test a website?` -> `Load testing a website`

Reply:
I want to this a FAQ.   Not including.